doc/basic_configuration.rst

```diff
@@ -100,3 +121,50 @@ repository::
 
 You can extract the repository PGP key from the ``buildfarm_deployment``
 configuration.
+
+
+Update URLs to point to required build tool repositories
```

Comment:
Please don't use the term "build tool" for the repository here. "build tool" has a different semantic meaning. Maybe just: "Update URLs to point to required repositories".

Reply:
Thanks! Fixed.